src/Sulu/Bundle/SnippetBundle/Content/SnippetContent.php

```diff
@@ -197,9 +197,13 @@ private function getSnippets(PropertyInterface $property)
         $snippetType = $this->getParameterValue($property->getParams(), 'snippetType');
         $default = $this->getParameterValue($property->getParams(), 'default', false);
 
-        if (empty($ids) && $snippetType && $default && $this->defaultEnabled) {
+        if ($default === true || $default === 'true') {
+            $default = $snippetType;
```

Comment:
$default is the default area e.g.: `sidebar.article` defined in the xml:

```xml
<property name="snippets" type="snippet">
    <meta>
        <title lang="en">Snippets</title>
    </meta>

    <params>
        <param name="snippetType" value="sidebar"/>
        <param name="default" value="sidebar.article"/>
    </params>
</property>
```

for the old behaviour:

```xml
        <param name="snippetType" value="sidebar"/>
        <param name="default" value="true"/>
```

for the old behaviour the default need to be set to the snippetType.

would you prefer something like:

```
        $snippetArea = $default;
        if ($snippetArea === true || $snippetArea === 'true') {
            $snippetArea = $snippetType;
        }

        if (empty($ids) && $snippetArea && $this->defaultEnabled) {
            $ids = [
                $this->defaultSnippetManager->loadIdentifier($webspaceKey, $snippetArea),
            ];
            // ...
``